test/packages/EABase/include/Common/EABase/eabase.h

```diff
@@ -114,8 +114,8 @@
 	   #define __STDC_FORMAT_MACROS
 	#endif
 	// The GCC PSP compiler defines standard int types (e.g. uint32_t) but not PRId8, etc.
-	// MSVC doesn't include an inttypes.h header.
-	#if !defined(EA_COMPILER_MSVC)
+	// MSVC added support for inttypes.h header in VS2013.
+	#if !defined(EA_COMPILER_MSVC) || EA_COMPILER_VERSION >= 1800
```

Comment:
Actually `defined(EA_COMPILER_MSVC)` would be redundant, because the right-hand side of the `||` operator would be evaluated only if `EA_COMPILER_MSVC` has already been defined.